src/status_im/ui/screens/signing/views.cljs

```diff
@@ -274,7 +282,8 @@
                   {:keys [amount-error gas-error]} [:signing/amount-errors (:address from)]
                   keycard-multiaccount? [:keycard-multiaccount?]
                   prices                [:prices]
-                  wallet-currency       [:wallet/currency]]
+                  wallet-currency       [:wallet/currency]
+                  window-width [:dimensions/window-width]]
```

Comment:
@acolytec3 and here we need to remove `window-width`

Reply:
Done.